automation/test.sh

```diff
@@ -390,6 +390,8 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
     fi
     if [[ $KUBEVIRT_WITH_MULTUS_V3 == "true" ]]; then
       add_to_label_filter "(!in-place-hotplug-NICs)" "&&"
+    else
```

Comment:
nit: There is a typo in the commit message and PR description:
> When hotplugging an interface to a VM running on a cluster will multus v3,

Should be:
> When hotplugging an interface to a VM running on a cluster **with** multus v3,

Reply:
done.